evals/registry/evals/pararule-plus-multi-step-deductive-reasoning.yaml

```
@@ -0,0 +1,7 @@
+pararule-plus-multi-step-deductive-reasoning:
+  id: pararule-plus-multi-step-deductive-reasoning.dev.v0
+  metrics: [accuracy]
+pararule-plus-multi-step-deductive-reasoning.dev.v0:
+  class: evals.elsuite.basic.match:FuzzyMatch
```

to use fuzzy match, the class should be:
```suggestion
  class: evals.elsuite.basic.fuzzy_match:FuzzyMatch
```